2021-05-09 12:39:36,760 INFO    MainThread:12876 [wandb_setup.py:_flush():69] Unhandled environment var: WANDB_LOG_MODEL
2021-05-09 12:39:36,760 INFO    MainThread:12876 [wandb_setup.py:_flush():69] Unhandled environment var: WANDB_WATCH
2021-05-09 12:39:36,760 INFO    MainThread:12876 [wandb_setup.py:_flush():69] setting env: {'silent': 'true'}
2021-05-09 12:39:36,760 INFO    MainThread:12876 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-09 12:39:36,760 INFO    MainThread:12876 [wandb_setup.py:_flush():69] [34m[1mwandb[0m: Currently logged in as: [33mdlrgy22[0m (use `wandb login --relogin` to force relogin)
2021-05-09 12:39:36,760 INFO    MainThread:12876 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-09 12:39:36,760 INFO    MainThread:12876 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/mycode/code/wandb/run-20210509_123936-1gqbh5x9/logs/debug.log
2021-05-09 12:39:36,760 INFO    MainThread:12876 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/mycode/code/wandb/run-20210509_123936-1gqbh5x9/logs/debug-internal.log
2021-05-09 12:39:36,760 INFO    MainThread:12876 [wandb_init.py:init():369] calling init triggers
2021-05-09 12:39:36,761 INFO    MainThread:12876 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-05-09 12:39:36,761 INFO    MainThread:12876 [wandb_init.py:init():418] starting backend
2021-05-09 12:39:36,761 INFO    MainThread:12876 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-09 12:39:36,829 INFO    MainThread:12876 [backend.py:ensure_launched():123] starting backend process...
2021-05-09 12:39:36,885 INFO    MainThread:12876 [backend.py:ensure_launched():128] started backend process with pid: 13034
2021-05-09 12:39:36,887 INFO    MainThread:12876 [wandb_init.py:init():423] backend started and connected
2021-05-09 12:39:36,889 INFO    MainThread:12876 [wandb_init.py:init():465] updated telemetry
2021-05-09 12:39:36,889 INFO    MainThread:12876 [wandb_init.py:init():484] communicating current version
2021-05-09 12:39:38,128 INFO    MainThread:12876 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.30 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-05-09 12:39:38,129 INFO    MainThread:12876 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-09 12:39:38,355 INFO    MainThread:12876 [wandb_init.py:init():522] starting run threads in backend
2021-05-09 12:39:41,106 INFO    MainThread:12876 [wandb_run.py:_console_start():1554] atexit reg
2021-05-09 12:39:41,107 INFO    MainThread:12876 [wandb_run.py:_redirect():1401] redirect: SettingsConsole.REDIRECT
2021-05-09 12:39:41,107 INFO    MainThread:12876 [wandb_run.py:_redirect():1406] Redirecting console.
2021-05-09 12:39:41,109 INFO    MainThread:12876 [wandb_run.py:_redirect():1468] Redirects installed.
2021-05-09 12:39:41,109 INFO    MainThread:12876 [wandb_init.py:init():546] run started, returning control to user process
2021-05-09 12:39:41,744 INFO    MainThread:12876 [wandb_run.py:_atexit_cleanup():1524] got exitcode: 1
2021-05-09 12:39:41,747 INFO    MainThread:12876 [wandb_run.py:_restore():1496] restore
2021-05-09 12:39:43,953 INFO    MainThread:12876 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 988
  total_bytes: 988
}

2021-05-09 12:39:44,371 INFO    MainThread:12876 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 988
  total_bytes: 4064
}

2021-05-09 12:39:44,753 INFO    MainThread:12876 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4066
  total_bytes: 4066
}

2021-05-09 12:39:44,854 INFO    MainThread:12876 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4066
  total_bytes: 4066
}

2021-05-09 12:39:44,956 INFO    MainThread:12876 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4066
  total_bytes: 4066
}

2021-05-09 12:39:45,057 INFO    MainThread:12876 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4066
  total_bytes: 4066
}

2021-05-09 12:39:45,158 INFO    MainThread:12876 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4066
  total_bytes: 4066
}

2021-05-09 12:39:45,260 INFO    MainThread:12876 [wandb_run.py:_wait_for_finish():1646] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4066
  total_bytes: 4066
}

